src/librustc_mir/const_eval/fn_queries.rs

```diff
@@ -116,21 +124,6 @@ fn is_const_fn_raw(tcx: TyCtxt<'_>, def_id: DefId) -> bool {
     }
 }
 
-/// Const evaluability whitelist is here to check evaluability at the
-/// top level beforehand.
```

Comment:
This comment is outdated. Where did the whitelist go? I don't remember removing it.

Reply:
This became outdated in 9ee16e14c51668309f446636e8960b7fbbac066d, which began using attributes on intrinsics.